types/permit/permit-tests.ts

```diff
@@ -0,0 +1,61 @@
+import { IncomingMessage, ServerResponse } from "http";
+import { Permit, Basic, Bearer } from 'permit';
+
+const permit = new Permit({
+    scheme: "some-scheme",
+    proxy: "some-proxy",
+    realm: "auth"
+});
+
+const basic = new Basic({
+    scheme: "some-scheme",
+    proxy: "some-proxy",
+    realm: "auth"
+});
+
+const bearer = new Bearer({
+    basic: 'username',
+    query: 'access_token',
+});
+
+function permitHandler(req: IncomingMessage, res: ServerResponse) {
+    const token = permit.check(req);
```

Comment:
I think the lint error comes from here. the lint rule is trying to stop you from assigning the return value of a void function since it will always be undefined.

Reply:
Ah, indeed. Thanks for pointing that out.